Source/CarthageKit/VersionFile.swift

```diff
@@ -40,7 +40,7 @@ extension CachedFramework: Decodable {
 
 struct VersionFile {
 	let commitish: String
-	let xcodeVersion: String
+	let xcodeVersion: String?
```

Comment:
@mdiep sounds reasonable. I've gone ahead and made a separate PR to do this, #1896